### PR TITLE
feat: correct release tagging for multi sdk repo

### PR DESCRIPTION
**Pull request by ryan-timothy-albert**

### CODE REVIEW

File: internal/git/releases.go [hunk 1]
```diff
@@ -18,11 +18,14 @@ import (
 //go:embed goreleaser.yml
 var tfGoReleaserConfig string
 
-func (g *Git) SetReleaseToPublished(version string) error {
+func (g *Git) SetReleaseToPublished(version, directory string) error {
 	if g.repo == nil {
 		return fmt.Errorf("repo not cloned")
 	}
 	tag := "v" + version
+	if directory != "" && directory != "." && directory != "./" {
```
Comment (LukeHagar) [inline internal/git/releases.go:26]: Can you walk me through how this is intended to be used in this context? we do want to use directories as tags?

Comment (ryan-timothy-albert) [reply internal/git/releases.go:26]: This is how it already works today. Not changing anything

Comment (ryan-timothy-albert) [reply internal/git/releases.go:26]: I'm just making sure reading a release that was already created doesn't fail

Comment (ryan-timothy-albert) [reply internal/git/releases.go:26]: see https://github.com/codatio/client-sdk-typescript/releases

**Review by chase-crumbaugh — APPROVED** [review]

**Pull request merged**